### ISSUE TITLE
Papyrus: Running Dash Cancel Bug

### ISSUE THREAD

**Issue by waspennator** [ikemen-engine/Ikemen-GO#1194]
### Describe the bug

For some reason when running into opponents with Papyrus's running dash, on Mugen it's supposed to automatically exit out of the running dash after hitting the opponent with it. But on Ikemen GO, Papyrus doesn't automatically exit out of it like he should.

### To Reproduce

1: Download Papyrus from here. (Char File is too big to upload here)
https://mugenarchive.com/forums/downloads.php?do=file&id=68426-papyrus-mega-update-fourthrhyme
2: Start a match with Papyrus and anyone.
3: Double Tap right to do the running dash into the opponent and watch him not stop.

### Expected behavior

Papyrus should automatically stop his running dash after hitting his opponent with it,

### Screenshots / Video

On Mugen: (Stopping after each hit)

https://user-images.githubusercontent.com/78768499/234181006-217c3e17-ba1c-438f-9634-8d648a0262d7.mp4

On Ikemen GO: (Not stopping after each hit)

https://user-images.githubusercontent.com/78768499/234181468-c7fcef0a-7d47-4bb8-a5ca-35e046d7d2c9.mp4

### Engine Version (or source code date)

Nightly Build 4/24/23

### Operating system

Windows

**Comment by potsmugen** [comment]
This happens because the char uses a non-standard Mugen version in the DEF file. It's fixed if you change "mugenversion = 6/14/2015" to (I assume) "mugenversion = 1.1".

Although it would be nice if Ikemen could recognize some years in that line and interpreted them as the corresponding version.

**Comment by potsmugen** [comment]
I was looking into implementing Mugen version date recognition, but there are so many ways authors could write it (YYYY/MM/DD, YY-MM-DD, etc) that we're probably better off just telling people to correct it.

I made a PR so that every time a Mugen character is loaded, the console will tell you which compatibility mode it is using.

EDIT: On top of it all, this character uses a Mugen version supposedly released in 2015, which doesn't exist. So a version date recognition would still not understand it.

**Comment by waspennator** [comment]
> I was looking into implementing Mugen version date recognition, but there are so many ways authors could write it (YYYY/MM/DD, YY-MM-DD, etc) that we're probably better off just telling people to correct it.
> 
> I made a PR so that every time a Mugen character is loaded, the console will tell you which compatibility mode it is using.
> 
> EDIT: On top of it all, this character uses a Mugen version supposedly released in 2015, which doesn't exist. So a version date recognition would still not understand it.

Turns out fourthrhyme put out a new version last year that added 1.0 to the mugenversion section so I guess its working now, I can close it if need be

**Comment by potsmugen** [comment]
Yeah. Realistically there's probably nothing to fix here on the engine side.
